src/main/java/org/openbase/bco/bcozy/controller/RegistrationController.java

```diff
@@ -108,32 +153,47 @@ private void register() throws InterruptedException {
             sessionManager.verifyMailAddress(mail.getText());
         } catch (VerificationFailedException ex) {
             mail.getStyleClass().add("text-field-wrong");
+            mailEmptyLabel.setVisible(true);
             return;
         }
 
         List<UnitConfig> groups = usergroupField.getCheckModel().getCheckedItems();
 
+        UserConfigType.UserConfig.Builder userConfigBuilder = UserConfigType.UserConfig.newBuilder();
```

Comment:
Its not necessary to store this instance if never used again.

Reply:
fixed